snow/consensus/snowball/binary_snowflake.go

```diff
@@ -11,29 +11,38 @@ func newBinarySnowflake(alphaPreference, alphaConfidence, beta, choice int) bina
 	return binarySnowflake{
 		binarySlush:     newBinarySlush(choice),
 		alphaPreference: alphaPreference,
-		alphaConfidence: alphaConfidence,
-		beta:            beta,
+		terminationConditions: []terminationCondition{
+			{
+				alphaConfidence: alphaConfidence,
+				beta:            beta,
+			},
+		},
+		confidence: make([]int, 1),
```

Comment:
I cannot do this here because the value is being constructed, so I do not have a reference to `terminationConditions`.

I will make this change in the follow up PR where the constructor passes in an argument including the set of termination conditions.